server/lib/models/channel.go

```diff
@@ -65,3 +112,97 @@ func ListChannel(dbif Dbif, req *proto.ChannelListRequest) ([]*proto.Model_Chann
 	}
 	return ret, nil
 }
+
+//takes about 90sec
+func bytesColumnEncode(v []byte) string {
+	result := make([]byte, 3 + 4*len(v))
+	result[0] = 'b'
+	result[1] = '\''
+	start := 2
+	for i := 0; i < len(v); i++ {
+		b := v[i]
+		result[start + 4 * i] = '\\'
+		result[start + 4 * i + 1] = 'x'
+		result[start + 4 * i + 2] = "0123456789abcdef"[b >> 4]
+		result[start + 4 * i + 3] = "0123456789abcdef"[b & 0xf]
+	}
+	result[start + 4 * len(v)] = '\''
+	return string(result)
+}
+func Locales() []string {
```

Comment:
__[gometalinter]__
* [golint] exported function Locales should have comment or be unexported